applications: nrf_desktop: Add user guide images

pdunaj opened the pull request:
Jira:DESK-900

Signed-off-by: Pawel Dunaj <pawel.dunaj@nordicsemi.no>

NordicBuilder commented:
**Internal CI Error.**

Python exception in `../tools/ci-tools/scripts/check_compliance.py`:

```
Traceback (most recent call last):
  File "../tools/ci-tools/scripts/check_compliance.py", line 1412, in main
    n_fails = _main(args)
  File "../tools/ci-tools/scripts/check_compliance.py", line 1376, in _main
    n_fails = report_test_results_to_github(suite)
  File "../tools/ci-tools/scripts/check_compliance.py", line 1150, in report_test_results_to_github
    "**Some checks failed. Please fix and resubmit.**\n\n" + comment)
  File "../tools/ci-tools/scripts/check_compliance.py", line 1193, in github_comment
    github_pr.create_issue_comment(msg)
  File "/usr/local/lib/python3.6/site-packages/github/PullRequest.py", line 416, in create_issue_comment
    input=post_parameters
  File "/usr/local/lib/python3.6/site-packages/github/Requester.py", line 264, in requestJsonAndCheck
    return self.__check(*self.requestJson(verb, url, parameters, headers, input, self.__customConnection(url)))
  File "/usr/local/lib/python3.6/site-packages/github/Requester.py", line 275, in __check
    raise self.__createException(status, responseHeaders, output)
github.GithubException.GithubException: 422 {'message': 'Validation Failed', 'errors': [{'resource': 'IssueComment', 'code': 'unprocessable', 'field': 'data', 'message': 'Body is too long (maximum is 65536 characters)'}], 'documentation_url': 'https://developer.github.com/v3/issues/comments/#create-a-comment'}

```

*Tip: The bot edits this comment instead of posting a new one, so you can check the comment's history to see earlier messages.*

pdunaj commented:
Hi @thst-nordic , do you know what can be this "Internal CI Error."?

pdunaj commented:
Hi @greg-fer , I have added new images and rebased. Can you have a look. Unfortunately after the last changes in Zephyr I cannot render the documentation locally, again :(

greg-fer commented on applications/nrf_desktop/README.rst:
```diff
@@ -585,23 +603,111 @@ Otherwise, the old version is used.
+
+   .. group-tab:: Keyboard
+
+      .. figure:: /images/nrf_desktop_keyboard_back.svg
```
**Image edit:**
Please remove the debug caption from this image.

greg-fer commented on applications/nrf_desktop/README.rst:
```diff
@@ -696,6 +808,13 @@ Hardware switch
 Precision Aim button
     * The button is located on the left side of the mouse, in the thumb area.
+
+      .. figure:: /images/nrf_desktop_gaming_mouse_top.svg
```
**Image edit:**
Please remove the debug caption from this image.

@pdunaj , do we need the mention of LED 1 on this image?
I think it should be OK to keep it, but without "- peer state", and with an explanation in the docs about this LED 1.

greg-fer commented on applications/nrf_desktop/README.rst:
```diff
@@ -708,25 +827,37 @@ Precision Aim button
       The scroll must be pressed before the mouse is powered up.
 
-        * |nRF_Desktop_confirmation_effect|
+      .. figure:: /images/nrf_desktop_desktop_mouse_side.svg
```
**Image edit:**
Please remove the debug caption from this image.

Also, @pdunaj, about middle button vs scroll wheel button. I think "scroll wheel button" is less confusing if the mouse has a button under the scroll. Some mice have DPI buttons located under the scroll wheel or above it, and they could also be considered "middle buttons".

greg-fer commented on applications/nrf_desktop/README.rst:
```diff
@@ -708,25 +827,37 @@ Precision Aim button
 Page Down key
     * The Page Down key must be pressed while keeping the Fn modifier key pressed.
+
+      .. figure:: /images/nrf_desktop_keyboard_top.svg
```
@pdunaj , do we need LED1 on this image? I think we can keep it, but would have to add explanation in a bullet point below.

greg-fer commented on applications/nrf_desktop/README.rst:
```diff
@@ -741,35 +872,58 @@ Page Down key
 SW1 button
+    * The SW1 button is located on the top of the dongle.
+
+      .. figure:: /images/nRF52840_dongle_front_callouts.svg
```
@pdunaj , same LED matter as for the keyboard and mouse. Let's keep the captions, but explain what the meaning of LEDs is.

greg-fer commented on applications/nrf_desktop/README.rst:
```diff
@@ -741,35 +872,58 @@ Page Down key
+.. tabs::
 
-The nRF Desktop peripherals are either suspended or powered off when they are not in use for a predefined amount of time:
+   .. group-tab:: Gaming mouse
```
**Image edit:**
Please remove all captions _except_ the debug one from this image, and from the following two images.

pdunaj commented:
Hi @greg-fer , please check if the images are ok. I have added few changes to text - please see if something needs modification.

pdunaj commented:
I have split the change to appease the CI...

pdunaj commented:
Splitting into multiple commits does not help the CI. We will need to move the images to the separate PRs. I will do it once we get review done.

pdunaj commented:
@thst-nordic , @wiminordic , do you know if checkpatch will ever finish? this is the pr where check crashed because of the large number of chars.

thst-nordic commented:
We have manually skipped the checkpatch, we can ignore it's status.

pdunaj commented:
Hi @carlescufi , could you merge it without the checkpatch? It won't ever finish because of the svg images.
